public/app/features/dashboard/components/PanelEditor/VisualizationSelectPane.tsx

```diff
@@ -74,7 +74,12 @@ export const VisualizationSelectPane: FC<Props> = ({ panel }) => {
 
   const radioOptions: Array<SelectableValue<ListMode>> = [
     { label: 'Visualizations', value: ListMode.Visualizations },
-    { label: 'Global panels', value: ListMode.Globals },
+    {
+      label: 'Global panels',
```

Comment:
```suggestion
      label: 'Library panels', 
```

This section should be Library panels, if I understood @jessover9000  recommendations.  Same for the tab under Dashboards section. But maybe something to discuss tomorrow.